tests/Testing/MockServerTestCase.php

```diff
@@ -56,26 +56,48 @@ protected function defaultUnpublishMessageJson($key): string
         return '{"key":"'.$key.'","action":"unpublish","eventTime":null,"properties":{},"payload":null}';
     }
 
-    protected function assertIngestionPostRequest(
+    protected function assertIngestionRequest(
         RequestInfo $request,
         string $uri,
         string $expectedBody,
         array $headers = null
     ): void {
-        $this->assertEquals('POST', $request->getRequestMethod());
+        $this->assertRequest($request, 'POST', $uri, $expectedBody, $headers);
+    }
+
+    protected function assertSchemaRequest(
+        RequestInfo $request,
+        string $uri,
+        array $headers = null
+    ): void {
+        $this->assertRequest($request, 'GET', $uri, null, $headers);
+    }
```

Comment:
Now both `assertIngestionRequest` and `assertSchemaRequest` use generic `assertRequest`